modules/administration/pages/auth-methods.adoc

```diff
@@ -2,99 +2,102 @@
+YaST can now be used to configure PAM, when packages such as [package]``yast2-ldap-client`` and [package]``yast2-kerberos-client`` are installed.
+For information about configuring PAM, see the SUSE Linux Enterprise Server Security Guide https://www.suse.com/documentation/sles-15/book_security/data/part_auth.html.
+This example is not limited to Kerberos; it is generic and uses the current server configuration.
+Only network based authentication services are supported.
 +
 .Changing Passwords
-IMPORTANT: Changing the password on the SUSE Manager Web interface changes only the local password on the SUSE Manager server.
-But this password may not be used at all if PAM is enabled for that user.
+IMPORTANT: Changing the password on the {productname} {webui} changes only the local password on the {productname} server.
+This password may not be used at all if PAM is enabled for that user.
 In the above example, for instance, the Kerberos password will not be changed.
-+
 
 
 
-== Authentication Via eDirectory and PAM
-
+=== Authentication Via eDirectory and PAM
 
 . First check to ensure eDirectory authentication is working with your current OS for example:
 +
 
 ----
-#getent passwd
-----
-. If users are returned from eDirectory then create the following file:
-+
-
-----
-# cat /etc/pam.d/susemanager
+getent passwd
 ----
-. And add the following content:
+. If users are returned from eDirectory then create the [filename]``/etc/pam.d/susemanager`` and add the following content:
 +
 
 ----
 #%PAM-1.0
- auth     include        common-auth
- account  include        common-account
- password include        common-password
- session  include        common-session
- #
+auth     include        common-auth
+account  include        common-account
+password include        common-password
+session  include        common-session
 ----
-. Finally add the following lines to the {productname} conf file:
+
+. Finally add the following line to the {productname} conf file [filename]``/etc/rhn/rhn.conf``:
 +
 
 ----
-# grep -i pam /etc/rhn/rhn.conf
- pam_auth_service = susemanager
+pam_auth_service = susemanager
 ----
-. You may now create users with the same id that appears on eDirectory and mark the Use PAM check-box from the SUSE Manager WebUI.
+
+You may now create users with the same id that appears on eDirectory and enable the [guimenu]``Use PAM`` check-box in the {productname} {webui}.
 
 
 
-== Example Quest VAS Active Directory Authentication Template
+=== Example Quest VAS Active Directory Authentication Template
```

Comment:
You could probably do this as `.Example:` rather than having a third-level heading.

Reply:
sounds good. done.